cypress/e2e/29-templates.cy.ts

```diff
@@ -31,4 +34,10 @@ describe('Templates', () => {
 		workflowPage.getters.stickies().should('have.length', 1);
 		workflowPage.actions.shouldHaveWorkflowName(OnboardingWorkflow.name);
 	});
+
+	it('can open template with images and hides workflow screenshots', () => {
```

Comment:
not the perfect test for this bug, but thought I would add a test here since this area is not covered at all